tests/tests_integration/test_api/test_simulators/test_simulators.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+
+from cognite.client import CogniteClient
+from cognite.client.data_classes.simulators.filters import SimulatorIntegrationFilter
+from tests.tests_integration.test_api.test_simulators.seed.data import (
+    simulator,
+    simulator_integration,
+    simulator_model,
+    simulator_model_revision,
+    simulator_routine,
+    simulator_routine_revision,
+)
+
+
+@pytest.fixture
+def add_simulator_resoures(cognite_client: CogniteClient) -> None:
+    simulator_external_id = "integration_tests_workflow"
+    simulator_model_file_external_id = "ShowerMixer_simulator_model_file"
+
+    file = cognite_client.files.upload(
+        path="tests/tests_integration/test_api/test_simulators/seed/data/ShowerMixer.dwxmz",
+        external_id=simulator_model_file_external_id,
+        name="ShowerMixer.dwxmz",
+        data_set_id=97552494921583,
+    )
```

Comment:
would be great to only upload if it doesn't exist and not to remove in the end of the test (slow operation)